src/components/Comments/Comment.js

```diff
@@ -42,6 +45,7 @@ const Comment = ({
   showLeftConnector = true,
   disableOnTextPress = false,
   onLongPress,
+  feedId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `feedId` has been introduced. Ensure that this prop is being passed correctly from the parent component and it's not null or undefined where it's required.



<!-- This is an auto-generated comment by OSS CodeRabbit -->